packages/neos-ui-editors/src/Editors/Range/index.js

```diff
@@ -41,8 +41,9 @@ class RangeEditor extends PureComponent {
     handleChange = event => {
         const {options} = this.props;
         const {target} = event;
+        const useParseInt = options.step % 1 === 0;
```

Comment:
clever idea with the steps ^^

```suggestion
        const useParseInt = (options.step || 1) % 1 === 0;
```

otherwise this would be false (if `step` is not defined) 

```
({}).step % 1 === 0
```

so but that means for full float support you have to set the step to stm like `step: 0.1`